proxy/src/main.rs

```diff
@@ -27,15 +27,15 @@ mod server_warp;
 mod source;
 
 fn main() {
-    std::env::set_var("RUST_BACKTRACE", "full");
```

Comment:
I'd like to keep that, esp. during our rapid development it's helpful, we can consider disabling it for release builds.

Reply:
I did not intend to commit this. Will revert.